elements/lisk-chain/src/data_access/storage.ts

```diff
@@ -417,11 +417,11 @@ export class Storage {
 		// Revert all deleted values
 		for (const { key, value: previousValue } of deletedStates) {
 			batch.put(key, previousValue);
-			await smt.update(toSMTKey(key), previousValue);
+			await smt.update(toSMTKey(key), hash(previousValue));
```

Comment:
SHould we rather use a util function `toSTMValue(value)` to be more consistent?

Reply:
`toSTMKey` has logic where it hashes excluding first 6 bytes, while for value it's just simple hash, so i think it is ok like this